Orange/widgets/data/owrank.py

```diff
@@ -198,9 +192,9 @@ class Outputs:
     sorting = Setting((0, Qt.DescendingOrder))
     selected_methods = Setting(set())
 
-    settings_version = 2
+    settings_version = 3
     settingsHandler = DomainContextHandler()
-    selected_rows = ContextSetting([])
+    selected_attrs = ContextSetting([], schema_only=True)
```

Comment:
Can `ContextSetting` can be `schema_only`?
I think it is always read from a context and ignores the schema_only flag.

At this point we should decide whether we want the selection as schema_only setting or context dependent.

Reply:
I would like it to be a schema-only context setting. :) If the flag is currently ignored, I'd still keep it for the case it is supported some day. But if you're sure it's ignored and you want to remove it, I can do so, too.